lib/gollum-lib/page.rb

```diff
@@ -88,6 +88,13 @@ def initialize(wiki)
       @parent_page    = nil
     end
 
+    # Public: The SHA hash identifying this page
+    #
+    # Returns the String SHA.
+    def sha
+      @blob && @blob.id
```

Comment:
Yeah, I noticed this pattern being used elsewhere in `page.rb`, too, so I decided to stick with the style. We could refactor it throughout the codebase if we don't like it. :)